h2o-algos/src/main/java/hex/rulefit/RuleFitModel.java

```diff
@@ -9,8 +9,12 @@
 import water.udf.CFuncRef;
 import water.util.TwoDimTable;
 
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
 
-public class RuleFitModel extends Model<RuleFitModel, RuleFitModel.RuleFitParameters, RuleFitModel.RuleFitOutput> {
+public class RuleFitModel extends Model<RuleFitModel, RuleFitModel.RuleFitParameters, RuleFitModel.RuleFitOutput> implements FitRulesCollector {
```

Comment:
One more note:

The R and Python API call the `predict_rules` method based on the discussion about naming, but in Java, you keep everywhere `FitRules`. Maybe it is only a detail, but I would keep the naming the same everywhere. 

WDYT @koniecsveta ?

Reply:
yesyes, will rename also in Java

Reply:
b615b24